data/json/mutations/mutations.json

```diff
@@ -3804,7 +3804,12 @@
     "points": 1,
     "vitamin_cost": 60,
     "visibility": 1,
-    "enchantments": [ { "melee_damage_bonus": [ { "type": "stab", "add": 0.5 } ] } ],
+    "enchantments": [
+      {
+        "condition": { "and": [ { "not": "u_has_weapon" }, { "math": [ "u_encumbrance('hand_l') + u_encumbrance('hand_r') <= 0" ] } ] },
```

Comment:
My knowledge of the game might be outdated but there were fingerless gloves which would illogically block this mutation from working as they add encumbrance

Reply:
Imo i would personally removed this enchantment at all, it's too small to be used at all